.github/workflows/pr-checks.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           - 2.13.6
```

Comment:
Could you remove this line? We’re only camping for the last three versions of 2.13.

Reply:
Removed, thanks for the feedback